include/nuttx/wireless/gs2200m.h

```diff
@@ -116,6 +116,7 @@ struct gs2200m_recv_msg
   FAR uint8_t *buf;
   uint16_t    len;    /* actual buffer length */
   uint16_t    reqlen; /* requested size */
+  int32_t     flags;  /* MSG_* flags */
```

Comment:
should we have code manipulate the new field?

Reply:
Yes, but not now. I want to leave it to @masayuki2009 .

This patch is needed for prevent build break by daemon patches (in apps), I'll create it after this patch is getting merged.

Reply:
@SPRESENSE, @xiaoxiang781216 

The field will be used in the future.